examples/internal/cluster.tf

```diff
@@ -0,0 +1,74 @@
+resource "aws_ecs_cluster" "example" {
+  name = "${var.service}-${var.env}"
+
+  tags = {
+    Name = "${var.service}-${var.env}"
+  }
+
+  setting {
+    name  = "containerInsights"
+    value = "enabled"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      setting,
+    ]
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution_role" {
```

Comment:
I'm getting a clash on a `make build` with the resource here and within `container-proxy.tf` on L73 - can this be removed as the `ecs_task_execution_variable` is set to `null`?

Reply:
...Or have the examples set that `ecs_task_execution_variable` accordingly to something similar to `${var.service}-${var.env}`